src/codegen/ruby/def/event.rb

```diff
@@ -1,52 +0,0 @@
-
-require_relative 'field'
-
-class EventDef
```

Comment:
Why are we removing this?

Reply:
Instead the ResourceDef class is used. They were very close to identical, with the minor differences now set by configuration.